SPECS/kubernetes/kubernetes-1.17.spec

```diff
@@ -0,0 +1,205 @@
+%global debug_package %{nil}
+%ifarch x86_64
+%define archname amd64
+%endif
+%ifarch aarch64
+%define archname arm64
+%endif
+Summary:        Microsoft Kubernetes
+Name:           kubernetes
+Version:        1.17.11
+Release:        3%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Microsoft Kubernetes
+URL:            https://mcr.microsoft.com/oss
+#               Download URL https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.11-hotfix.20200901/binaries/kubernetes-node-linux-amd64.tar.gz
```

Comment:
Our convention with instances where the SourceX url doesn't perfectly match the source name is to put a `"#SourceX: url"` above it. In this case, it'd be replacing the "Downloading URL" with:

`#Source0: https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.11-hotfix.20200901/binaries/kubernetes-node-linux-amd64.tar.gz`

Applies to the other SPECs changed in this PR as well.

Reply:
I didn't know that, I'll fix it